motor/core.py

```diff
@@ -90,6 +90,11 @@ def __repr__(self):
 
 
 class AgnosticBaseProperties(AgnosticBase):
+    # Allow the use of generics at runtime.
+    @classmethod
+    def __class_getitem__(cls, key: str) -> object:
+        return cls
```

Comment:
Should this raise NotImplemented?

Reply:
No,

```python
 python test/check_runtime_types.py
Traceback (most recent call last):
  File "/Users/steve.silvester/workspace/motor/test/check_runtime_types.py", line 12, in <module>
    client: AsyncIOMotorClient[Dict[str, Any]]
            ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^
  File "/Users/steve.silvester/workspace/motor/motor/core.py", line 96, in __class_getitem__
    raise NotImplementedError
NotImplementedError
```

Reply:
Actually never mind, that would defeat the whole purpose.